model/src/main/java/io/quarkus/vault/runtime/client/dto/kv/VaultKvSecretV2WriteBody.java

```diff
@@ -2,11 +2,43 @@
 
 import java.util.Map;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import io.quarkus.vault.runtime.client.dto.VaultModel;
 
 public class VaultKvSecretV2WriteBody implements VaultModel {
```

Comment:
These DTO classes are internal and will most be generated at some point in the future. No need to clean them up or change them at this point.